ocs_ci/utility/localstorage.py

```diff
@@ -128,6 +128,29 @@ def check_local_volume():
         return "No resources found" not in status
 
 
+def check_local_volume_set():
```

Comment:
this function is almost identical to `check_local_volume`. How about creating a more generic function that gets the resource type and the command to replace both functions?

Reply:
Done

Reply:
@prsurve I meant to have 1 generic function instead of `check_local_volume()` and `check_local_volume_set()`

Reply:
I Will try to collocate both the things in single function

Reply:
Done